newIDE/app/src/Utils/GDevelopServices/Build.js

```diff
@@ -80,7 +80,10 @@ export const getWebBuildThumbnailUrl = (
   }
   // The exporter put asset files directly in the build folder.
   // It's not factorized with the exporter because it's a temporary solution.
-  return `https://games.gdevelop-app.com/game-${buildId}/${fileName}`;
+  const uri = `https://games.gdevelop-app.com/game-${buildId}/${fileName}`;
+  // The backend services encode the file URLs when uploaded, so we need to do the same before saving the value.
```

Comment:
It's a case where you expect the same behaviour between frontend and backend. I wonder if it is worth having unit tests for that, so that when changing it in the future, you really ask yourself if this has some impact

Reply:
Ok, maybe just add a TODO then